SU-SWS/stanford_profile
# D8CORE-850 Ckeditor Sticky Toolbar

# READY FOR REVIEW

# Summary
- Enables sticky toolbar module

# Need Review By (Date)
- 12/11

# Urgency
- low

# Steps to Test
1. `compose require su-sws/stanford_profile:dev-D8CORE-850-ckeditor-sticky --prefer-source`
1. `drush cim -y` or `blt drupal:install`
1. create a page with a wysiwyg
1. add a ton of text
1. scroll down and verify the toolbar sticks to the top of the window.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
